test/audits/audio-without-controls-test.js

```diff
@@ -0,0 +1,14 @@
+module("Audio");
+
+test("Audio has no controls attribute.", function() {
+  // Setup fixture
+  var fixture = document.getElementById('qunit-fixture');
+  fixture.innerHTML = "<audio src=\"http://developer.mozilla.org/@api/deki/files/2926/=AudioTest_(1).ogg\" autoplay></audio>";
+  var audio = document.querySelector('audio');
+  audio.addEventListener('canplay', function() {
```

Comment:
As discussed, we need to come up with a way to block this test finishing until this assertion has run. Check out the QUnit doc: http://api.qunitjs.com/